internal/services/storage/storage_account_resource.go

```diff
@@ -403,6 +403,12 @@ func resourceStorageAccount() *pluginsdk.Resource {
 							Default:  false,
 						},
 
+						"change_feed_retention_in_days": {
```

Comment:
It should be more like other retention_policies
example
```
	"delete_retention_policy": {
							Type:     pluginsdk.TypeList,
							Optional: true,
							MaxItems: 1,
							Elem: &pluginsdk.Resource{
								Schema: map[string]*pluginsdk.Schema{
									"days": {
										Type:         pluginsdk.TypeInt,
										Optional:     true,
										Default:      7,
										ValidateFunc: validation.IntBetween(1, 365),
									},
								},
							},
						},
```

Reply:
No its 365 and default is 7
![image](https://user-images.githubusercontent.com/3121149/174896112-f6ec0fc4-0e29-4a82-a1ba-9c41e9bfc18b.png)

Reply:
This is a bug in Portal, the API can use a larger value. Also the default value used in TF is not necessarily needed to be the same as Portal.